plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/metadata/TestDatabaseShardManager.java

```diff
@@ -694,14 +678,9 @@ public void testMaintenanceBlocked()
         dbi.onDemand(MetadataDao.class).blockMaintenance(tableId);
 
         long transactionId = shardManager.beginTransaction();
-        try {
-            shardManager.replaceShardUuids(transactionId, tableId, columns, oldShards, ImmutableSet.of(), OptionalLong.empty());
-            fail("expected exception");
-        }
-        catch (TrinoException e) {
-            assertEquals(e.getErrorCode(), TRANSACTION_CONFLICT.toErrorCode());
-            assertEquals(e.getMessage(), "Maintenance is blocked for table");
-        }
+        assertTrinoExceptionThrownBy(() -> shardManager.replaceShardUuids(transactionId, tableId, columns, oldShards, ImmutableSet.of(), OptionalLong.empty()))
+                .hasErrorCode(TRANSACTION_CONFLICT)
+                .hasMessage("Maintenance is blocked for table");
     }
```

Comment:
same comment as above: shall we put long transactionId = shardManager.beginTransaction(); inside lambda?

Reply:
@MariyaR I've refactored the code as suggested.